upgrading/topics/keycloak/changes.adoc

```diff
@@ -257,7 +269,7 @@ we cannot assume which one will be used to fulfill the auth request so the reque
 ==== Upgrade to Wildfly 15
 
 The {project_name} server was upgraded to use Wildfly 15 as the underlying container. This does not directly involve any
-specific {project_name} server functionality, but there are few changes related to the migration, which worth mentioning.
+specific {project_name} server functionality, but there are few changes related to the migration, which are worth mentioning.
```

Comment:
change this to "but a few configuration changes related to..."

Reply:
Corrected per the suggestion. Thanks, Andy